pkgs/top-level/all-packages.nix

```diff
@@ -12717,7 +12717,7 @@ in
       inherit stdenv fetchurl fetchFromGitHub makeSetupHook;
       bison = bison2; # error: too few arguments to function 'int yylex(...
       inherit cups;
-      harfbuzz = harfbuzzFull;
+      inherit harfbuzz;
```

Comment:
This should be under `qt512` in order for `qtbase` to pass `configurePhase`.

Reply:
Fixed!